lang/ilias_fa.lang

```diff
@@ -1177,7 +1177,7 @@ chatroom_adm#:#client_settings#:#تنظیمات کلاینت
 chatroom_adm#:#enable_osd#:#فعال کردن آگاه سازی روی صفحه
 chatroom_adm#:#general_settings_title#:#تنظیمات کلی چت
 chatroom_adm#:#https#:#HTTPS
-chatroom_adm#:#osd_intervall#:#بازه راه اندازی مجدد، ثانیه
+chatroom_adm#:#osd_interval#:#بازه راه اندازی مجدد، ثانیه
```

Comment:
Please only touch the `de` and `en` language files. This change has to be reverted.